apps/betterangels-backend/accounts/models.py

```diff
@@ -123,18 +131,46 @@ class Meta:
 class ClientProfile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="client_profile")
     address = models.TextField(blank=True, null=True)
+    city_of_birth = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
city of birth sounds like "Los Angeles", not "Los Angeles, CA", i think place is better because then people can write it more generic things like "Ukraine"
```suggestion
    place_of_birth = models.CharField(max_length=100, blank=True, null=True)
```

Reply:
good call